specs/subresourceintegrity/spec.markdown

```diff
@@ -576,7 +576,7 @@ as possible.
 <section>
 ##### HTMLLinkElement
 
-attribute DOMString integrity
+readonly attribute DOMString integrity
```

Comment:
We have it as `readonly` in Section 3.4 (modifications to Fetch) too.

I thought that was to indicate that it can only be read and never modified. I can't think of a reason why we'd want to modify it. Are you thinking perhaps of add-ons being able to change it?